lib/between_meals/changes/change.rb

```diff
@@ -24,13 +24,22 @@ module BetweenMeals
   module Changes
     # Common functionality
     class Change
+      ALLOWED_STATUSES = [:modified, :deleted].freeze
```

Comment:
```suggestion
      # Since we either need to upload or delete, we only accept two statuses.
      # VCSs will differentiate between various kinds of modifies, adds, etc.
      # so instead of handling all possibilities here, we expect the caller to
      # collapse them into `:modified` or `:deleted`.
      ALLOWED_STATUSES = [:modified, :deleted].freeze
```